test/rbac.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    kubernetes.io/metadata.name: cm-tests
+  name: cm-tests
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cert-manager-operator-e2e-test-runner
+  namespace: cm-tests
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: cert-manager-operator-e2e
+subjects:
+  - kind: ServiceAccount
+    name: cert-manager-operator-e2e-test-runner
+    namespace: cm-tests
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
I'm not sure what is the biggest permissions are needed. I suggest code uses least needed permissions. For example, if the "admin" role on namespace cm-tests for the SA is enough, I suggest using only "admin" role on namespace cm-tests.

Reply:
Since we need to manage pods in another namespace, create ClusterIssuers and other non namespaced resources, I went  with using cluster-admin, without it ClusterIssuer creation was failing